postStart.sh

```diff
@@ -7,42 +7,37 @@ until curl --output /dev/null --silent --head --fail http://${HOST}/; do
   sleep 5
 done
 
+
 chgrp -R 0 /nexus-data
 chmod -R g+rw /nexus-data
 find /nexus-data -type d -exec chmod g+x {} +
 
 NEXUS_BASE_DIR="/opt/sonatype/nexus"
-NEXUS_REPO_DIR="${NEXUS_BASE_DIR}/repositories"
+NEXUS_MAVEN_REPO_DIR="${NEXUS_BASE_DIR}/maven-proxy-repositories"
+NEXUS_NPMJS_REPO_DIR="${NEXUS_BASE_DIR}/npmjs-proxy-repositories"
 USERNAME="admin"
 PASSWORD="admin123"
 PASSWORD_FROM_FILE="$(cat "${NEXUS_BASE_DIR}"/config/password || true)"
```

Comment:
I am at loss how who or what would be writing to ${NEXUS_BASE_DIR}"/config/password? I have a feeling this script is broken to start with.

Reply:
Any chance you looked at some of the startup logs for this charts? I am starting to wonder whether the password never gets changed in the first place.

Reply:
I see, the password gets mounted in via the secret.